packages/core/core/src/Transformation.js

```diff
@@ -201,12 +210,15 @@ export default class Transformation {
     );
 
     // TODO: sort
-    let configHashesId = JSON.stringify(
-      [...configs].map(([, config]) => config.resultHash)
+    let configsId = JSON.stringify(
+      [...configs].map(([, {resultHash, devDeps}]) => ({
```

Comment:
nit: maybe a list of tuples rather than a list of objects which lack guaranteed key order

Reply:
I think I should probably just build one big object and call `md5FromObject` which guarantees a sorted key order.